src/foam/u2/view/ScrollTableView.js

```diff
@@ -141,11 +150,10 @@
       name: 'onWheel',
       code: function(e) {
         var negative = e.deltaY < 0;
-        // Convert to rows, rounding up. (Therefore minumum 1.)
-        var rows = Math.ceil(Math.abs(e.deltaY) / 40);
-        var oldSkip = this.skip;
+        var rows = Math.floor(Math.abs(this.accumulator + e.deltaY) / this.rowHeight);
+        this.accumulator += e.deltaY;
         this.skip = Math.max(0, this.skip + (negative ? -rows : rows));
-        if ( e.deltaY !== 0 && oldSkip != this.skip ) e.preventDefault();
+        if ( e.deltaY !== 0 ) e.preventDefault();
```

Comment:
Would it be better if we just always prevented default?

Reply:
That would prevent us from scrolling horizontally.